src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,67 @@
-export const Tabs = () => {};
+import React from 'react';
+import classNames from 'classnames';
+
+import { Tab } from '../../types/Tab';
+
+type Props = {
+  tabs: Tab[],
+  selectedTabId: string,
+  onTabSelected(tab: Tab): void,
```

Comment:
```suggestion
  onTabSelected(tab: Tab): void,
```
create type for your function separately (for me it looks better)